modules/packages/UnorderedCopy.chpl

```diff
@@ -22,10 +22,16 @@
      This module represents work in progress. The API is unstable and likely to
      change over time.
 
-   This module provides an unordered version of copy/assign for ``numeric``
-   types. The results from this function are not visible until task or forall
-   termination or an explicit :proc:`unorderedCopyTaskFence()`, but it can
-   provide a significant speedup for bulk assignment operations that do not
+   This module provides an unordered version of copy/assign for trivially
+   copyable types types. Trivially copyable types are types that can be
+   duplicated by copying bits. They include ``numeric`` and ``bool`` types as
+   well as ``numeric``/``bool`` tuples and ``numeric``/``bool`` records
+   provided the record has no user-defined initializers, deinitializers, or
+   assignment overloads.
+
+   The results from :proc:`unorderedCopy()`, are not visible until task or
+   forall termination or an explicit :proc:`unorderedCopyTaskFence()`, but it
```

Comment:
Let's make the two halves of this sentence into different sentences and maybe even different paragraphs (fencing and performance seem almost totally different topics to me)